olive/model/hf_utils.py

```diff
@@ -78,8 +78,8 @@ class HFModelLoadingArgs(ConfigBase):
     extra_args: Dict = Field(
```

Comment:
This can be removed.

Reply:
I kept this since I wanted to override the description of the field. The behavior remains the same.